.vscode/tasks.json

```diff
@@ -0,0 +1,62 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
```

Comment:
Removing these two lines of comments resolves the JSON syntax warning `Comments are not permitted in JSON.(521)`

```suggestion
```

Reply:
Ok, removed.